### PR TITLE
WIP: Adding support for s390x

### PR DESCRIPTION
**Pull request by salamani**
PR for #2880

### CODE REVIEW

**Comment by dlorenc** [comment]
Sorry, I don't think we can support builds for other architectures like this without a CI setup that can run tests. We've discussed this a few times for other architectures: 
https://github.com/kubernetes/minikube/pull/2455

**Comment by salamani** [comment]
@dlorenc  Could you provide more details about CI setup? We would like to join CI setup. Please let us know the steps to follow.

**Comment by salamani** [comment]
Any update?

**Comment by salamani** [comment]
Any updates ?

**Comment by guirish** [comment]
Hi @salamani @dlorenc any updates on this PR w.r.t s390x inclusion in CI ?

**Comment by tstromberg** [comment]
> Hi @salamani @dlorenc any updates on this PR w.r.t s390x inclusion in CI ?

We don't yet have a reasonable way of including s390x into the current CI system, but I would be willing to accept this port and list it as experimental for the time being. This situation may chance once our CI environment moves to Prow (#3256)

File: deploy/addons-s390x/addon-manager.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
```
[the remark this answers is not in the document]
Comment (salamani) [reply deploy/addons-s390x/addon-manager.yaml]: Yes . I agree, I am finding a way to templatize it. Let me know if you have any suggestion.

**Comment by tstromberg** [comment]
This looks great! Do you mind rebasing? I think it's about where we can accept this.

**Comment by salamani** [comment]
@tstromberg  I am creating new PR as I am getting multiple conflicts during rebase

**Comment by afbjorklund** [comment]
These image templates are getting uglier and uglier, think we need another approach here...

**Comment by tstromberg** [comment]
Dupe of #4091

**Pull request closed without merging**